lib/event_sourcery/memory/projector.rb

```diff
@@ -0,0 +1,25 @@
+module EventSourcery
+  module Memory
+    module Projector
+
+      def self.included(base)
+        base.include(EventSourcery::EventProcessing::EventStreamProcessor)
+        base.include(InstanceMethods)
+        base.class_eval do
+          alias_method :project, :process
+          class << self
+            alias_method :project, :process
+            alias_method :projects_events, :processes_events
+            alias_method :projector_name, :processor_name
+          end
+        end
```

Comment:
Oh, there's no `Projector` either in ES!
That's exactly what I meant when I said I was feeling uncomfortable about the split.

I think both `Projector` and `Reactor` should be ES classes where you can inject in Postgres or In Memory support.